src/jack.c

```diff
@@ -318,44 +318,56 @@ jalv_backend_init(Jalv* jalv)
 {
 	jack_client_t* client = NULL;
 
```

Comment:
It'd be nice if this was restructured so the existing code was in another function at the same indentation level so the diff was less noisy, if easily possible.

If not, whatever.